dataprovider/src/main/java/cm/aptoide/pt/dataprovider/ws/v7/ListSearchAppsRequest.java

```diff
@@ -79,8 +91,8 @@ public static ListSearchAppsRequest of(String query, int offset, boolean addSubs
 
   @Override protected Observable<ListSearchApps> loadDataFromNetwork(Interfaces interfaces,
       boolean bypassCache) {
-    return interfaces.listSearchApps(body, bypassCache,
-        appBundlesVisibilityManager.shouldEnableAppBundles());
+    return interfaces.listSearchApps(bypassCache ? "no-cache" : null,
```

Comment:
bypassCache is always passed as true, so there doesn't seem to be any gain with this. As we've discused, you might want to try and pass the value as false and see if there's any difference (see SearchManager@searchAppInStores)